satellite/metainfo/metainfo.go

```diff
@@ -1158,6 +1163,33 @@ func (endpoint *Endpoint) GetObject(ctx context.Context, req *pb.ObjectGetReques
 
 	if pointer.Remote != nil {
 		object.RedundancyScheme = pointer.Remote.Redundancy
+	} else if streamMeta.NumberOfSegments == 0 || streamMeta.NumberOfSegments > 1 {
+		// workaround
+		// new metainfo API redundancy scheme is on object level (not per segment) because
+		// of that RS is taken always from last segment, old implemetation were saving RS
+		// per segment and in some cases when for remote file last segment is inline segment
+		// then we are missing RS. This part will search for RS in othere segments than last one
+
+		// NumberOfSegments == 0 - pointer with encrypted num of segments
+		// NumberOfSegments > 1 - pointer with unencrypted num of segments and multiple segments
```

Comment:
maybe these two comments can go above the check

Reply:
done